### PR TITLE
qc housing stat link

### PR DESCRIPTION
**Pull request by velrest**
- feat(global): add landing page
- feat(quarterly-closure): add fist closure status handling
- feat(quarterly-closure): add error list to quarterly closure
- feat(quarterly-closure): add link to housing stat for maintenance cost

### CODE REVIEW

**Comment by velrest** [comment]
depends on #306

**Comment by velrest** [comment]
image

File: addon/components/quarterly-closure/error-list/survey-department-errors.hbs [hunk 1]
```diff
@@ -1,4 +1,19 @@
 <h3>{{t "ember-gwr.quarterlyClosure.components.departmentErrors.title"}}</h3>
+
+{{#if this.isFirstQuarter}}
```
Comment (luytena) [inline addon/components/quarterly-closure/error-list/survey-department-errors.hbs:3]: Use quarter in progress (not depending on current date)

File: translations/quarterly-closure/de.yaml [hunk 2]
```diff
@@ -56,3 +56,4 @@ ember-gwr:
         error: Beim Abrufen der Fehlerliste der Projekte ist ein Fehler aufgetreten.
         title: Erhebungsstelle
         infoText: Nachfolgend finden Sie die offenen Pendenzen der Erhebungsstelle. Für einen erfolgreichen Quartalsabschluss müssen mindestens alle Fehler vom Typ ADR oder BAU behoben werden.
+        maintenanceCost: Die Ausgaben für den Unterhalt an eigenen Bauwerken müssen im 1. Quartal im Housing Stat gesetzt werden.
```
Comment (luytena) [inline translations/quarterly-closure/de.yaml:59]: Housing-Stat?

**Review by luytena — APPROVED** [review]

**Pull request merged**